homeassistant/components/daikin/climate.py

```diff
@@ -150,7 +150,7 @@ async def _set(self, settings):
             # temperature
             elif attr == ATTR_TEMPERATURE:
                 try:
-                    values[HA_ATTR_TO_DAIKIN[ATTR_TARGET_TEMPERATURE]] = str(int(value))
+                    values[HA_ATTR_TO_DAIKIN[ATTR_TARGET_TEMPERATURE]] = str(value)
                 except ValueError:
```

Comment:
I expect you can't generate a value error any more, so probably could remove this.

But without the int(), aren't you allowing in anything, including bad data?  Looking at the the pydaikin API, it seems that whatever you pass in here will end up in the URL of the API call.  So perhaps convert to float and maybe round to precision 0.1 before passing it?  Any invalid data would then be trapped here.

Reply:
Yes you are right. I pushed a new commit with the suggested change.